style-guide/src/pages/otkit-desktop-typography.js

```diff
@@ -1,66 +1,71 @@
-import React from 'react';
-import Link from 'gatsby-link';
-import _ from 'lodash';
+import React from "react";
```

Comment:
these should be single quotes everywhere, please. Also, I know this isn't your fault, because we don't have a linter yet, and these kinds of rules should be encoded by the linter. Sorry >_<

Reply:
We used prettier with default settings...sorry let me change the setting. We can definitely automate later on with pre-commit prettier...

Reply:
yeah, that sounds good 👍